src/Windows.Core/NTStatusServerity.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <summary>
+    /// Represent the 4 high bits of a <see cref="NTStatus"/> enumeration : Severity (2 bits), Customer code (1 bit) and Reserved (1 bit).
+    /// </summary>
+    internal static class NTStatusServerity
+    {
+        public const uint Success = (uint)0xC << 28;
```

Comment:
Avoid using visibility modifiers that exceed the visibility of the containing type. Otherwise it looks like these contribute to the public API of the library when in fact they don't.